### PR TITLE
FAI-9231 | Partial Vuln Fix - curl CVE-2023-38545 

### PR DESCRIPTION
**Pull request by omreego**
## Description
Updating Node version and Alpine version and fixes curl vuln
https://nvd.nist.gov/vuln/detail/CVE-2023-38545

## Type of change
- [x] Bug fix
- [x] New feature
- [ ] Breaking change

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM node:16-alpine
+FROM node:21-alpine
```
Comment (tovbinm) [inline Dockerfile:1]: before we do this, we need to update the node version across the whole repo and make sure it compiles & runs - for example https://github.com/faros-ai/airbyte-connectors/pull/542

**Review by tovbinm — CHANGES_REQUESTED** [review]
not as simple :( see - https://github.com/faros-ai/airbyte-connectors/pull/542

File: Dockerfile (hunk 1 above)
Comment (omreego) [reply Dockerfile:1]: Ah, good to know.
I'll try out the steps in #542

**Comment by tovbinm** [comment]
I am not sure if nodejs is fully backward compatible. Folks, how can we test this?

**Pull request closed without merging**